flatpak/qgis.json

```diff
@@ -0,0 +1,118 @@
+{
+    "id": "org.qgis.qgis",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "master",
```

Comment:
You should probably use the stable kde runtime version 5.9 which is available on flathub, rather than the unstable master version.